### PR TITLE
Update notes on self signed certificates for desktop app

### PR DESCRIPTION
**Pull request by MikesGlitch**
Notes on mkcert when used with desktop app. 

Should resolve future questions like this one: https://discord.com/channels/937901803608096828/1281484103580909621/1284432537682317365

### CODE REVIEW

File: docs/config/https.md [hunk 1]
```diff
@@ -39,5 +39,16 @@
...
 If you can’t easily create new files, you can also configure HTTPS using environment variables. Set the `ACTUAL_HTTPS_KEY` and `ACTUAL_HTTPS_CERT` environment variables to the contents of the `.key` and `.crt` files, respectively. If you’re unable to include newlines in the environment variable values, you can replace any newlines with `\n` and Actual will automatically convert them back to newlines.
 
+### Configuring the Desktop App: 
+To configure the desktop app, follow these steps on the "Where's the server?" screen. 
+
+1. Enter your server URL and click OK.
+2. If you see a error message asking you to select your certificate, choose the appropriate certificate and try again.
+
+**Note:** If you have generated your certificate using [mkcert](https://github.com/FiloSottile/mkcert) you need to specify the root CA certificate - not the certificate your server is pointing to. To do this:
+1. Run the command ```mkcert -CAROOT``` to find the directory of root certificate.
```
[the remark this answers is not in the document]
Comment (MikesGlitch) [reply docs/config/https.md:49]: @check-spelling-bot apply [updates](https://github.com/actualbudget/docs/actions/runs/10861180477/attempts/1).

[the remark this answers is not in the document]
Comment (MikesGlitch) [reply docs/config/https.md:49]: @check-spelling-bot apply [updates](https://github.com/actualbudget/docs/actions/runs/10861180477/attempts/1)

**Comment by MikesGlitch** [comment]
@check-spelling-bot apply [updates](https://github.com/actualbudget/docs/actions/runs/10861180477/attempts/1).

File: docs/config/https.md [hunk 2]
```diff
@@ -39,5 +39,16 @@ If using a Docker container, this folder is `/data` within the container. If you
 ### Configuring with environment variables: 
 If you can’t easily create new files, you can also configure HTTPS using environment variables. Set the `ACTUAL_HTTPS_KEY` and `ACTUAL_HTTPS_CERT` environment variables to the contents of the `.key` and `.crt` files, respectively. If you’re unable to include newlines in the environment variable values, you can replace any newlines with `\n` and Actual will automatically convert them back to newlines.
 
+### Configuring the Desktop App: 
+To configure the desktop app, follow these steps on the "Where's the server?" screen. 
+
+1. Enter your server URL and click OK.
+2. If you see a error message asking you to select your certificate, choose the appropriate certificate and try again.
```
Comment (RubenOlsen) [inline docs/config/https.md:46]:
```suggestion
2. If you see an error message asking you to select your certificate, choose the appropriate certificate and try again.
```

File: docs/config/https.md [hunk 3]
```diff
@@ -39,5 +39,16 @@ If using a Docker container, this folder is `/data` within the container. If you
...
+
+**Note:** If you have generated your certificate using [mkcert](https://github.com/FiloSottile/mkcert) you need to specify the root CA certificate - not the certificate your server is pointing to. To do this:
```
Comment (RubenOlsen) [inline docs/config/https.md:48]: - Comma before you
- need to => must

Or maybe this suggestion (shorter sentence) is ok with you:

```suggestion
**Note:** If you have generated your certificate using [mkcert](https://github.com/FiloSottile/mkcert), specify the root CA certificate - not the certificate your server is pointing to. To do this:
```

File: docs/config/https.md [hunk 4]
```diff
@@ -39,5 +39,16 @@ If using a Docker container, this folder is `/data` within the container. If you
 ### Configuring with environment variables: 
 If you can’t easily create new files, you can also configure HTTPS using environment variables. Set the `ACTUAL_HTTPS_KEY` and `ACTUAL_HTTPS_CERT` environment variables to the contents of the `.key` and `.crt` files, respectively. If you’re unable to include newlines in the environment variable values, you can replace any newlines with `\n` and Actual will automatically convert them back to newlines.
 
+### Configuring the Desktop App: 
+To configure the desktop app, follow these steps on the "Where's the server?" screen.
```
Comment (RubenOlsen) [inline docs/config/https.md:43]: One should avoid putting anything in quotes unless referencing speech or being sarcastic :-)

It is better to use _italics_ for this.

```suggestion
To configure the desktop app, follow these steps on the _Where's the server?_ screen. 
```

**Review by RubenOlsen — CHANGES_REQUESTED** [review]
A few nit-pics.

**Comment by MikesGlitch** [comment]
> A few nit-pics.

Updated, thanks for insight on the quotes. 👍

**Comment by RubenOlsen** [comment]
You are welcome :-)

One could also have included the appropriate screenshot - to make things even more precise.

In general, use italics to reference fields, etc - and bold to reference content (in the fields).

**Review by RubenOlsen — APPROVED** [review]

**Pull request merged**